pinot-core/src/main/java/org/apache/pinot/core/operator/BaseOperator.java

```diff
@@ -35,7 +35,7 @@ public final T nextBlock() {
     if (Thread.interrupted()) {
       throw new EarlyTerminationException();
     }
-    try (InvocationScope execution = Tracing.getTracer().createScope(getClass())) {
+    try (InvocationScope ignored = Tracing.getTracer().createScope(getClass())) {
```

Comment:
calling this `ignored` is kind of missing the point and will mislead the reader, it's more than just ceremony but a scope. I think they should be called scope throughout, except when referenced as recordings (so that they can't be closed without casting)

Reply:
Changed it because the IDE is putting a warning on unused variable. Another way to suppress the warning would be adding a comment as following if that is preferred:
```
    //noinspection unused
    try (InvocationScope scope = Tracing.getTracer().createScope(getClass())) {
      return getNextBlock();
    }
```

Reply:
2nd option seems better but upto you